source/component/PasDoc_Items.pas

```diff
@@ -2844,12 +2844,19 @@ procedure TPasRoutine.Serialize(const ADestination: TStream);
 end;
 
 function TPasRoutine.Signature: string;
-begin
-  ParamTypes.QuoteChar := #0;
-  ParamTypes.Delimiter := ',';
 
+  function GetParamTypesStr: string;
+  var
+    ParamType: string;
+  begin
+    Result := '';
+    for ParamType in ParamTypes do
+      Result := IfThen(Result = '', '', Result + ',') + SRemoveChars(ParamType, [' ']);
+  end;
+
+begin
   if ParamTypes.Count > 0 then
-    Result := Name + '(' + SRemoveChars(ParamTypes.DelimitedText, [' ']) + ')'
+    Result := Format('%s(%s)', [Name, GetParamTypesStr])
```

Comment:
Add here a comment like 

```
{ Note: We're not using TStringList.DelimitedText because of FPC 3.2.0 bug
  https://gitlab.com/freepascal.org/fpc/source/-/issues/37605 
}
```